components/sst_importer/src/sst_importer.rs

```diff
@@ -348,6 +349,129 @@ impl SSTImporter {
     pub fn list_ssts(&self) -> Result<Vec<SstMeta>> {
         self.dir.list_ssts()
     }
+
+    pub fn new_writer<E: KvEngine>(
+        &self,
+        default: E::SstWriter,
+        write: E::SstWriter,
+        meta: SstMeta,
+    ) -> Result<SSTWriter<E>> {
+        // use new uuid to generate different default and write filename
+        let mut default_meta = meta.clone();
+        let cf = CF_DEFAULT.to_owned();
+        default_meta.set_cf_name(cf);
+        let default_path = self.dir.join(&default_meta)?;
+
+        let mut write_meta = meta;
+        let cf = CF_WRITE.to_owned();
+        write_meta.set_cf_name(cf);
+        let write_path = self.dir.join(&write_meta)?;
+        Ok(SSTWriter::new(
+            default,
+            write,
+            default_path,
+            write_path,
+            default_meta,
+            write_meta,
+        ))
+    }
+}
+
+pub struct SSTWriter<E: KvEngine> {
```

Comment:
this probably doesn't need to be `pub`.

Reply:
we will build SSTWriter in src/import/sst_services.rs, so this need to be public.

Reply:
Okay